templates/repo/branch_dropdown.tmpl

```diff
@@ -50,13 +50,13 @@
 		'enableFeed': {{.root.EnableFeed}},
 		'rssURLPrefix': '{{$.root.RepoLink}}/rss/branch/',
 		'branchURLPrefix': '{{if .branchURLPrefix}}{{.branchURLPrefix}}{{else}}{{$.root.RepoLink}}/{{if $.root.PageIsCommits}}commits{{else}}src{{end}}/branch/{{end}}',
-		'branchURLSuffix': '{{if .branchURLSuffix}}{{.branchURLSuffix}}{{else}}{{if $.root.TreePath}}/{{PathEscapeSegments $.root.TreePath}}{{end}}{{end}}',
+		'branchURLSuffix': '{{if .branchURLSuffix}}{{.branchURLSuffix}}{{else}}{{if ctx.Locale.TreePath}}/{{PathEscapeSegments $.root.TreePath}}{{end}}{{end}}',
```

Comment:
I do not feel safe to use your regexp. What's this change?

And I guess there are more problems.

Reply:
Okay, yeah, I can guess what happened here.
Will update my regexp.

Reply:
I think I've fixed all edge cases now.